src/register/register-gnome/gnucash-item-edit.c

```diff
@@ -333,6 +332,32 @@ unblock_toggle_signals(GncItemEdit *item_edit)
 }
 
 
+static gboolean
+draw_arrow_cb (GtkWidget *widget, cairo_t *cr, gpointer data)
```

Comment:
Interesting one this, in the other locations the arrow is only required to be set in one direction so it would not work in this location.